From 5a310d66411986b48f020f921e57f3b48ce8d725 Mon Sep 17 00:00:00 2001
From: cleanbaja <circutrider21@outlook.com>
Date: Tue, 5 Apr 2022 14:40:39 -0500
Subject: [PATCH] Hide {open,stat}64 definitions from mkinit

---
 src/mkinit.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/mkinit.c b/src/mkinit.c
index 9714bee..b76e1fe 100644
--- a/src/mkinit.c
+++ b/src/mkinit.c
@@ -43,6 +43,14 @@
  */
 
 
+/* NOTE(cleanbaja): since mlibc dosen't support
+ * {open, stat, etc}64 family functions, 
+ * but linux does, undefine the macros...
+ */
+
+#undef open64
+#undef stat64
+
 #include <sys/types.h>
 #include <stdio.h>
 #include <stdlib.h>
-- 
2.25.1

